### DISCUSSION TITLE
Add 2.0 studio style

### DISCUSSION THREAD

**Discussion by JackyBorderCollie** [ScratchAddons/ScratchAddons#2686] [Ideas]
When the new 3.0 studio style will be released, you could add an option in Scratch Addons to bring back 2.0 style studio. It will be good for Scratchers, who want to use the old style.

**Comment by daniel4-scratch** [comment]
I would like some time traveling.

**Comment by mxmou** [comment]
I don't think it would make sense to make an addon like this specifically for studio pages. It's an interesting idea though.

**Reply by WorldLanguages** [reply]
FYI, 3.0→2.0 is highly suggested through feedback.

**Reply by mxmou** [reply]
I'd like to work on it once I finish the website dark mode rewrite (probably not very soon).

**Reply by JackyBorderCollie** [reply]
Cool!
